public/app/features/logs/components/panel/processing.ts

```diff
@@ -97,8 +98,23 @@ export class LogListModel implements LogRowModel {
     this.raw = raw;
   }
 
+  clone() {
+    const clone = Object.assign(Object.create(Object.getPrototypeOf(this)), this);
```

Comment:
This has been created only for LogLineDetailsLog, so for now let's accept this corner-cutting of creating a wrapped clone so new lines are not stripped. If highlighting is toggled off, we render the unmodified entry, which has been thought for people doing stuff like having megabytes-long log lines that may not want highlighting. Other cases can be considered based on feedback or concrete examples.